src/ALObject.ts

```diff
@@ -252,18 +334,19 @@ export class ALObject {
                         break;
                 }
                 if (!skipToken) {
-                    if (xliffIdWithNames.length > 0 && (xliffIdWithNames[xliffIdWithNames.length - 1].IsMlToken || xliffIdWithNames[xliffIdWithNames.length - 1].Type === newToken.Type)) {
+                    if (xliffIdWithNames.length > 0 && (xliffIdWithNames[xliffIdWithNames.length - 1].isMlToken || xliffIdWithNames[xliffIdWithNames.length - 1].type === newToken.type)) {
                         xliffIdWithNames.pop();
                     }
                     xliffIdWithNames.push(newToken);
                 }
             }
-            const MlTokenPattern = /(OptionCaption|Caption|ToolTip|InstructionalText|PromotedActionCategories|RequestFilterHeading) = '.*'/i;
+            const MlTokenPattern = /(OptionCaption|Caption|ToolTip|InstructionalText|PromotedActionCategories|RequestFilterHeading) = '(.*?)'/i;
             const LabelTokenPattern = /(\w*): (Label) '.*'/i;
-
+            const ObjectPropertyTokenPattern = /(SourceTable|PageType) = (.*);/i;
```

Comment:
So is it PascalCasing for `const` and camelCasing for `let` and `var`? Since you've made a lot of changes pertaining to casing maybe we should add a link or docs for coding guidelines?

Reply:
Nope, just my search & replace that failed...